kuma/users/apps.py

```diff
@@ -13,3 +15,11 @@ class UserConfig(AuthConfig):
     def ready(self):
         # Connect signal handlers
         from . import signal_handlers  # noqa
+
+        # Configure global 'stripe' module
+        if settings.STRIPE_SECRET_KEY:
+            # XXX Perhaps we ought to sanity check that *all* other Stripe
+            # related settings are properly set.
```

Comment:
Let's not do that. It's out of context. It's perhaps weird but you might have a `STRIPE_SECRET_KEY` but prefer to not set up the `STRIPE_PLAN_ID`.